### ISSUE TITLE
Dynamic compression?

### ISSUE THREAD

**Issue by Synthematophon** [Archie3d/aeolus_plugin#33]
I have noticed another phenomenon: on a real organ, the increase in volume is not linear as the number of stops is increased. The sound "thickens" or becomes more intense rather than just getting louder. I don't know how to explain this better. Maybe it's just a psychoacoustic phenomenon. Nevertheless, would it be possible to introduce some kind of dynamic compression or something like that? So that when playing with an 8' flute the sound is not quite so quiet or weak, and when playing tutti you are almost overwhelmed by the volume.

**Comment by Archie3d** [comment]
Indeed. This is related to a spatialisation that happens naturally with real organs, which is difficult to achieve by digital summation of individual pipes signals into stereo. If you have access to a multichannel audio system (like 5.1 or 7.1) there is a multichannel (8 channels) build of the plugin/standalone you may try - it will gradually "fill" the surround space with sound achieving the effect you are talking about. The plugin internally already performs a binaural spatialisation of the pipes, I suppose this can be improved because currently there is no flexible way to position individual ranks of pipes and the default arrangement I've implemented perhaps is suboptimal.
 
Speaking of stereo, indeed, a limiter may help to get that non-linear volume increase without clipping, but this will also introduce some distortion (i.e. unnatural harmonics will appear), I am not sure how bad this can be, need to experiment.

**Comment by Archie3d** [comment]
Merged to develop.
